src/content/docs/en/release/diff-fedora.mdx

```diff
@@ -33,13 +40,13 @@ Changes listed below are nonexhaustive and cumulative, meaning each section cont
 - IP MTU probing enabled (which fixes [Ubisoft Connect](https://en.wikipedia.org/wiki/Ubisoft_Connect) games)
 - [Custom wallpapers](https://github.com/Ultramarine-Linux/backgrounds) 🌇 by Aikoyori
 - 📦 [System76 Scheduler](https://github.com/pop-os/system76-scheduler) (significantly better scheduling for low-end devices)
-    - 📦 [KWin Script for system76-scheduler Integration](https://github.com/maxiberta/kwin-system76-scheduler-integration)
+  - 📦 [KWin Script for system76-scheduler Integration](https://github.com/maxiberta/kwin-system76-scheduler-integration)
 - Numerous popular GNOME extensions
-    - 🛳️ [AppIndicator/KStatusNotifierItem support](https://github.com/ubuntu/gnome-shell-extension-appindicator) (`gnome-shell-extension-appindicator`)
-    - 🛳️ [Pop shell extension](https://github.com/pop-os/shell) (`gnome-shell-extension-pop-shell`)
-    - 🛳️ Support for keyboard selection of windows and workspaces in GNOME Shell (`gnome-shell-extension-windowsNavigator`)
-    - 🛳️ [Bringing back the app menu](https://github.com/fthx/appmenu-is-back) (`gnome-shell-extension-appmenu-is-back`)
-    - 📦 [Pop Launcher](https://github.com/pop-os/launcher) (`pop-launcher`)
+  - 🛳️ [AppIndicator/KStatusNotifierItem support](https://github.com/ubuntu/gnome-shell-extension-appindicator) (`gnome-shell-extension-appindicator`)
+  - 🛳️ [Pop shell extension](https://github.com/pop-os/shell) (`gnome-shell-extension-pop-shell`)
+  - 🛳️ Support for keyboard selection of windows and workspaces in GNOME Shell (`gnome-shell-extension-windowsNavigator`)
+  - 🛳️ [Bringing back the app menu](https://github.com/fthx/appmenu-is-back) (`gnome-shell-extension-appmenu-is-back`)
+  - 📦 [Pop Launcher](https://github.com/pop-os/launcher) (`pop-launcher`)
```

Comment:
?

Reply:
prettier ran on this

Reply:
I use prettier too, can relate… please revert

Reply:
will do